jmclient/jmclient/wallet_service.py

```diff
@@ -73,14 +86,36 @@ def update_blockheight(self):
         but will be called as part of main monitoring
         loop to ensure new transactions are added at
         the right height.
+        Any failure of the RPC call must result in this returning
+        False, otherwise return True (means self.current_blockheight
+        has been correctly updated).
         """
+
+        def critical_error():
```

Comment:
Rename this to `get_blockheight_error()`?

Reply:
I mean it's scoped only within the function, so I'd feel it's fine even to use `error()`. I think I'll leave it, just to avoid another commit.